.github/labeler.yml

```diff
@@ -62,6 +62,9 @@
 'category: docs_snippets':
 - 'docs/snippets/**/*'
 
+'md_files_only':
+- all: ['**/*.md']
```

Comment:
looks like we can easily check it by file extensions for file in change set and avoid extra helper label